app/views/planning_applications/validation/other_change_validation_requests/_form.html.erb

```diff
@@ -35,7 +35,7 @@
           class: "govuk-label govuk-label--s",
           text: (t(".labels.summary") if @validate_fee)
         },
-        hint: { class: ("make-invisible" if @validate_fee) },
+        hint: { class: ("display-none" if @validate_fee) },
```

Comment:
Should we be using `govuk-!-display-none`?

Reply:
Arguably, perhaps. I'm not very familiar with the gov.uk design system, but if we can remove more one-liner classes that seems good.

Reply:
That's prompted me to find a few other cases where we don't need to create our own utility classes.